hack/generate/catalog.sh

```diff
@@ -155,4 +155,8 @@ EOF
   fi
 }
 
+logger.info "Generating ImageContextSourcePolicy"
+create_image_content_source_policy "registry.ci.openshift.org/knative/${CURRENT_VERSION_IMAGES}:serverless-index" "$registry_redhat_io" "$registry_quay" "olm-catalog/serverless-operator-index/image_content_source_policy.yaml"
```

Comment:
This file is not applied anywhere. Will this be applied in some other repository?

Reply:
Yes, it's mostly for development and demos like https://github.com/keventmesh/eventing-platform-architecture